chapters/welcome.ipynb

```diff
@@ -29,28 +29,69 @@
     "docker compose up\n",
     "```\n",
     "\n",
-    "The book's code examples work with many different search engines and vector databases. By default, the book uses Apache Solr as the primary storage, matching, and ranking engine, but you can easily swap in your your favorite engine by running the `set_engine` function here: [engines/supported-engines.ipynb](engines/supported-engines.ipynb).\n",
-    "\n",
-    "If you find the examples not running as expected. Please ensure you have built the entire AI-Powered Search project and are running it per the instructions at:\n",
-    "http://github.com/treygrainger/ai-powered-search\n"
+    "The book's code examples work with many different search engines and vector databases. By default, the book uses Apache Solr as the primary storage, matching, and ranking engine, but you can easily swap in your your favorite engine by running the `set_engine`"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Run Healthcheck:"
+    "### Set a search engine:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'engines.solr.SolrEngine.SolrEngine'>\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
I don't think wee want to include a cell that sets the engine in the welcome.ipynb file. This might be the reader's first time running things, and if they are running OpenSearch already, this will just break things when they go to try to do their healthcheck, or else require them to modify the code or know to skip over these cells.

I think we should either:
1) Move this explanation to the engines Readme instead of the welcome.ipynb (originally, this just linked to the Readme for instructions, I think), or
2) If we keep this here, we should comment the code out so it doesn't run unless the person running it intentionally modifies the code (which means they know what they're doing or at least think they do).

Since the right engine _should_ already be set based on the Docker command they ran to start, this makes the "set_engine()" function effectively an advanced feature (to swap between engines when they user is running multiple engines at the same time). As such, I'm leaning towards NOT including this in the welcome and just referring them to the documentation.

Reply:
I've removed this from the welcome and verified it's sufficiently documented in the engine readme.